webtau-feature-testing/src/main/groovy/com/twosigma/webtau/featuretesting/WebTauEndToEndTestRunner.groovy

```diff
@@ -62,15 +56,15 @@ class WebTauEndToEndTestRunner implements StepReporter, StandaloneTestListener {
         StepReporters.add(this)
         StandaloneTestListeners.add(this)
 
-        def reportPath = 'examples/'
-        if (testFileName.endsWith('.groovy')) {
-            reportPath += testFileName.replace('.groovy', '-webtau-report.html')
-        } else {
-            reportPath += testFileName + '/webtau-report.html'
-        }
+        def targetClasses = DocumentationArtifactsLocation.classBasedLocation(WebTauEndToEndTestRunner)
```

Comment:
minor naming: this is a path but the variable name implies a collection, maybe rename to `targetClassesLocation` or `pathToTargetClasses` or some such (avoiding the term `classPath`)

Reply:
good point. renaming.